msteams-platform/tabs/what-are-custom-tabs.md

```diff
@@ -6,40 +6,48 @@ ms.topic: overview
+## Tab content requirements
 
-* Allow themselves to be iFramed by Teams (via the X-Frame-Options and/or Content-Security-Policy headers). A lot of standard webpages don't allow themselves to be iFramed which is why there is the option for users to view Website tab instances inside of a webview within the desktop client. Other tabs don't get this special treatment.
-* Handle [authentication](foo.md) differently (either via a popup or calling us to fetch tokens). Most websites simply redirect to a login provider which typically dead ends tabs which are hosted inside of an iframe. That's because login pages typically don't render in iFrames to prevent click-jacking.
-* Handle [cross-domain](foo.md) navigation differently since the Teams client needs to validate the origin against a static validDomains list in the app manifest when loading or communicating with the tab.
-* Style themselves based on the Teams client's theme.
-* On page load, call `microsoftTeams.initialize()` from the [Teams client SDK](/javascript/api/overview/msteams-client), which gives Teams a communication channel with the hosted page and more visibility into its operations.
+>&#x2705; You must allow your pages to be served in a nested browsing context by Teams in an HTML inline frame (IFrame) via X-Frame-Options and/or Content-Security-Policy HTTP response headers.
+<br><br>Many standard websites block their pages from loading in an IFrame. However, Teams provides a unique webview option for users to view embedded content within the desktop client.
 
-## Tabs on mobile clients
+>&#x2705; Your [authentication](foo.md) logic must leverage a method other than redirect (e.g., use token-based or cookie-based authentication).
+<br><br>Typically, as a safeguard against click-jacking, login pages don't render in IFrames.
 
-Teams is xPlat, your app needs to as be as well.
+>&#x2705; Your [cross-domain](foo.md) navigation logic should utilize methods that allow the Teams client to validate the origin against a static validDomains list in the app manifest when loading or communicating with the tab.
+<br><br>Browsers adhere to a same-origin policy restriction that prevents a webpage from making requests to a different domain than the one that served a web page. However, you may need to redirect the configuration or content page to a another domain or subdomain.
 
-contentUrl vs websiteUrl.
+>&#x2705; Style your tabs based on the Teams client's theme, design, and intent (see [Content and conversations, all at once using tabs](foo.md)).
+<br><br>Tabs work best when they're built to address a specific need and focus on a small set of tasks or a subset of data that is relevant to the tab's channel location.
+
+>&#x2705; Following your page load, make a call to `microsoftTeams.initialize()` in the [Teams client SDK](/javascript/api/overview/msteams-client).
+<br><br>Your script must include the microsoftTeams.initialize() method call for your page to be displayed.
```

Comment:
I like the green checkboxes for this, but not the '>' effect, or the line breaks. Try using a bulleted list w/o a line break for the explanation sentence. So something like:

&#x2705; This is the thing you must do.
* This is why we make you do it.

Overall though, I like this format for providing explanation for why something is required, and I'll copy it elsewhere where appropriate.